tests/components/system_bridge/test_init.py

```diff
@@ -37,6 +38,7 @@ async def test_migration_minor_1_to_2(hass: HomeAssistant) -> None:
         CONF_PORT: FIXTURE_USER_INPUT[CONF_PORT],
         CONF_TOKEN: FIXTURE_USER_INPUT[CONF_TOKEN],
     }
+    assert config_entry.state == ConfigEntryState.SETUP_RETRY
```

Comment:
We should patch the library to avoid I/O and so we can check that the integration is setup correctly. I'd expect the config entry state to be `ConfigEntryState.LOADED` here.